Adjust type for total score fields

nanaya opened the pull request:
Seems working. At least for the index schema creation, no idea if it fixes the query problem ("failed to create query: Value [3552068564] is out of range for an integer").

Resolves #157.

peppy commented:
I think this would only need to apply for `legacy_total_score`.. but maybe we just want to limit the values at the indexer level instead, since the *old `osu_score_high` tables don't support this either* (and i hope they won't ever have to and we move forward with lazer in cases where this matters).

nanaya commented:
I didn't check the score closer but apparently the one I'm looking at was a cheater score?

I've updated the schema to just legacy score and added pr to limit total score to 2B on osu-web.

peppy commented:
this will require an index rebuild, right? not sure if it's worth it when this has only happened once ever, like we probably want to guard this from reaching ES.

web-10 just doesn't allow scores this high, so it's likely a new component which is.

nanaya commented:
I don't know if it ever happened with legacy total score; the ones I saw are lazer scores. If there's a safety check somewhere to prevent legacy total score from hitting then this change isn't needed yeah. I don't think clamping it for indexing makes sense though if the number is legitimate as it would potentially fuck up leaderboard.

It can still be merged and just never applied until next index rebuild anyway.
